### PR TITLE
[AS9726-32D] support show system-health

### PR DESCRIPTION
**Pull request by seanwu-ec**
#### Why I did it
support `show system-health` on AS9726-32D model.

#### How I did it
Give a SystemHealth config json, ensure all required systemLED APIs are implemented, Fan, PSU info can be fetched correctly.

#### How to verify it
Make sure all Hardware checks are passed. 
Note : PSU1 wasn't powered on DUT.
```
root@as9726-32d-2:/home/admin# show system-health detail
System status summary

  System status LED  ControlledByFPGA
  Services:
    Status: OK
  Hardware:
    Status: OK

System services and devices monitor list

Name                        Status    Type
--------------------------  --------  ----------
as9726-32d-2                OK        System
rsyslog                     OK        Process
root-overlay                OK        Filesystem
var-log                     OK        Filesystem
routeCheck                  OK        Program
diskCheck                   OK        Program
container_memory_telemetry  OK        Program
FAN-1F                      OK        Fan
FAN-1R                      OK        Fan
FAN-2F                      OK        Fan
FAN-2R                      OK        Fan
FAN-3F                      OK        Fan
FAN-3R                      OK        Fan
FAN-4F                      OK        Fan
FAN-4R                      OK        Fan
FAN-5F                      OK        Fan
FAN-5R                      OK        Fan
FAN-6F                      OK        Fan
FAN-6R                      OK        Fan
PSU-1 FAN-1                 OK        Fan
PSU-2 FAN-1                 OK        Fan
PSU 2                       OK        PSU

System services and devices ignore list

Name               Status    Type
-----------------  --------  -------
container_checker  Ignored   Service
psu.temperature    Ignored   Device
PSU 1              Ignored   Device
asic               Ignored   Device
```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [x] 202106

#### Description for the changelog

[Accton] support show system-health on AS9726-32D

#### A picture of a cute animal (not mandatory but encouraged)
Alt Text

### CODE REVIEW

**Comment by seanwu-ec** [comment]
> This pull request **introduces 1 alert** when merging [cf4aac3](https://github.com/Azure/sonic-buildimage/commit/cf4aac351b77a618803e9fea929f4952336b4053) into [302f889](https://github.com/Azure/sonic-buildimage/commit/302f889415b5e46be1ce2d83f248b9bc82a25fb2) - [view on LGTM.com](https://lgtm.com/projects/g/Azure/sonic-buildimage/rev/pr-b1c0ca5e549ae789c28631e38a32139de0da4a32)
> 
> **new alerts:**
> 
> * 1 for Wrong name for an argument in a call

I don't know how come LGTM got this alert. namedtuple DOES have 'defaults' argument, and it works well.

https://docs.python.org/3.7/library/collections.html#collections.namedtuple
https://github.com/python/cpython/blob/041bfafeb00b0e5e565986b01c00bf9cddce3b4c/Lib/collections/__init__.py#L316

**Comment by seanwu-ec** [comment]
Hi @lguohan, when you have a moment, would you tell me what changes I need to make in order to get this PR reviewed and merged? Appreciate it.

File: device/accton/x86_64-accton_as9726_32d-r0/pmon_daemon_control.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "skip_ledd": true,
-    "skip_thermalctld": true
+    "skip_pcied": true
```
Comment (sujinmkang) [inline device/accton/x86_64-accton_as9726_32d-r0/pmon_daemon_control.json:3]: Is there any specific reason to skip the pcied?

Comment (akenliu) [reply device/accton/x86_64-accton_as9726_32d-r0/pmon_daemon_control.json:3]: There is no pcie devices, so we skip.

**Comment by seanwu-ec** [comment]
sync code to the new PR#14237 due to the code conflict.
so close this.

**Pull request closed without merging**